testdata/wasmexport-noscheduler.go

```diff
@@ -9,6 +11,7 @@ func callTestMain()
 
 func main() {
 	// main.main is not used when using -buildmode=c-shared.
+	time.Sleep(time.Millisecond)
```

Comment:
Leftover debug? Otherwise a comment would be nice.

Reply:
Fixed! It's actually part of the test, I've added a comment.